index.html

```diff
@@ -26,13 +26,13 @@
                 </div>
                 <ul id="link">
                     <li class="mobile-link">
-                        <a href="#">Portfolio</a>
+                        <a href="#works">Portfolio</a>
                     </li>
                     <li class="mobile-link">
-                        <a href="#">About</a>
+                        <a href="#about">About</a>
                     </li>
                     <li class="mobile-link">
-                        <a href="#">Contact</a>
+                        <a href="#contact">Contact</a>
                     </li>
                 </ul>
             </div>
```

Comment:
### Figma with correction
![Figma (copy)](https://user-images.githubusercontent.com/93039359/181242368-1849dd11-37d9-4774-8652-66cebc3e508e.png)

- [ ] Please add this black bar in your design to follow Figma the template & to make your design look professional :blush: